File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCSACTDefaultJearyoInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class OCSACTDefaultJearyoInfo
    {
        private String _setHangmogCode;
        private String _suryang;
        private String _danui;
        private String _danuiName;

        public String SetHangmogCode
        {
            get { return this._setHangmogCode; }
            set { this._setHangmogCode = value; }
        }

        public String Suryang
        {
            get { return this._suryang; }
            set { this._suryang = value; }
        }

        public String Danui
        {
            get { return this._danui; }
            set { this._danui = value; }
        }

        public String DanuiName
        {
            get { return this._danuiName; }
            set { this._danuiName = value; }
        }

        public OCSACTDefaultJearyoInfo() { }

        public OCSACTDefaultJearyoInfo(String setHangmogCode, String suryang, String danui, String danuiName)
        {
            this._setHangmogCode = setHangmogCode;
            this._suryang = suryang;
            this._danui = danui;
            this._danuiName = danuiName;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCSACTDefaultJearyoInfo.java
package nta.med.data.model.ihis.ocso;

public class OCSACTDefaultJearyoInfo {
	private String setHangmogCode ;
	private String suryang          ;
	private String danui            ;
	private String danuiName       ;
	public OCSACTDefaultJearyoInfo(String setHangmogCode, String suryang,
			String danui, String danuiName) {
		super();
		this.setHangmogCode = setHangmogCode;
		this.suryang = suryang;
		this.danui = danui;
		this.danuiName = danuiName;
	}
	public String getSetHangmogCode() {
		return setHangmogCode;
	}
	public void setSetHangmogCode(String setHangmogCode) {
		this.setHangmogCode = setHangmogCode;
	}
	public String getSuryang() {
		return suryang;
	}
	public void setSuryang(String suryang) {
		this.suryang = suryang;
	}
	public String getDanui() {
		return danui;
	}
	public void setDanui(String danui) {
		this.danui = danui;
	}
	public String getDanuiName() {
		return danuiName;
	}
	public void setDanuiName(String danuiName) {
		this.danuiName = danuiName;
	}
	

}
